pkg/specgen/generate/container.go

```diff
@@ -26,9 +26,14 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 	var inspectData *libimage.ImageData
 	var err error
 	if s.Image != "" {
-		newImage, _, err = r.LibimageRuntime().LookupImage(s.Image, nil)
-		if err != nil {
-			return nil, err
+		newImage, _ := s.GetImage()
```

Comment:
```suggestion
		newImage, _ = s.GetImage()
```
I think you want to assign this to the outer scope.

Reply:
nice eye @Luap99

Reply:
Good catch.  I'll move the users of `newImage` up.  The potential of shadowing (that I ran into) is dangerous.